cirq-google/cirq_google/calibration/phased_fsim.py

```diff
@@ -247,11 +247,19 @@ class PhasedFSimCalibrationResult:
             quibts a and b either only (a, b) or only (b, a) is present.
         gate: Characterized gate for each qubit pair. This is copied from the matching
             PhasedFSimCalibrationRequest and is included to preserve execution context.
+        options: The options used to gather this result.
+        initial_fids: The circuit fidelities by depth before characterization of the PhasedFSim.
+            Not all characterization techniques can report this.
+        final_fids: The circuit fidelities by depth after characterization of the PhasedFSim.
+            Not all characterization techniques can report this.
+
     """
 
     parameters: Dict[Tuple[Qid, Qid], PhasedFSimCharacterization]
     gate: Gate
     options: PhasedFSimCalibrationOptions
+    initial_fids: Optional[pd.DataFrame] = None
```

Comment:
I just noticed that this object is not JSON-serializable which is actually the most important one to serialize! We need to add serialization but because of that I don't think we should use pd.DataFrame. Shouldn't this be a plain `Dict[Tuple[Qid, Qid], Dict[int, float]]`?

As for `pd.DataFrame`, I'd suggest to have a method get_fidelities_frame() which would return frame with columns [`qubit_a`, `qubit_b`, `depth`, `initial_fidelity`, `final_fidelity`] out of the underlying data.

Reply:
Sorry, this class is JSON-serializable, it was just cluttered but github diff. Could you add support for JSON serialization of the two new fields as well?

Reply:
Removed these fields for now. Will introduce in a follow-on PR where we can do it more thoughtfully.